File: Object-Oriented Programming/Códigos/Códigos/POO 2/Projeto Veiculos/Motocicleta.java

package Proj_Veiculos;

public class Motocicleta extends Veiculo{
    //Atributos.
    public String modelo;
    public String marca;
    
    public Motocicleta() {
    	
    }
    
    //Metodos Especiais
    public String getModelo() {
        return modelo;
    }

    public void setModelo(String modelo) {
        this.modelo = modelo;
    }

    public String getMarca() {
        return marca;
    }

    public void setMarca(String marca) {
        this.marca = marca;
    }
}

File: Programação Orientada a Objetos/Códigos/Códigos/POO 4/Veiculos/Motocicleta.php
<?php
require_once 'Veiculo.php';
class Motocicleta extends Veiculo {
	private $modelo;
	private $marca;

	public function getModelo(){
		return $this->modelo;
	}

	public function setModelo($modelo): void {
		$this->modelo = $modelo;
	}

	public function getMarca() {
		return $this->marca;
	}

	public function setMarca($marca): void {
		$this->marca = $marca;
	}
	
}